deeppavlov/contrib/skills/similarity_matching_skill/similarity_matching_skill.py

```diff
@@ -0,0 +1,84 @@
+from typing import Tuple, Optional, List
+import os
+
+from deeppavlov import train_model
+from deeppavlov import build_model
+from deeppavlov.core.commands.utils import expand_path
+from deeppavlov.core.common.log import get_logger
+from deeppavlov.core.skill.skill import Skill
+from deeppavlov.core.common.file import read_json
+from deeppavlov.configs import configs
+from deeppavlov.core.data.utils import update_dict_recursive
+
+log = get_logger(__name__)
+
+
+class SimilarityMatchingSkill(Skill):
+    """Skill, matches utterances to predefined phrases, returns corresponding response.
+
+    Allows to create skills that gives response to corresponding user's utterance
+    Skill returns response and confidence.
+
+    Args:
+        data_path: URL or local path to '.csv' file that contains two columns with Utterances and Responses.
+            User's utterance will be compared with Utterances column and response will be selected
+            from matching row from Responses column.
+        x_col_name: Name of the column in '.csv' file, that represents Utterances column.
+        y_col_name: Name of the column in '.csv' file, that represents Responses column.
+        save_load_path: Path, where model will be saved or loaded from. Chooses './similarity_matching' by default.
+        edit_dict: Dictionary of edits in config (has higher prior, than previous arguments).
+        train: Should model be trained or not.
+
+    Attributes:
+        model: Classifies user's utterance
+    """
+
+    def __init__(self, data_path: Optional[str] = None,
+                 x_col_name: Optional[str] = None, y_col_name: Optional[str] = None,
+                 save_load_path: Optional[str] = None, edit_dict: Optional[dict] = None, train: bool = True):
```

Comment:
```suggestion
                 save_load_path: str = './similarity_matching', edit_dict: Optional[dict] = None, train: bool = True):
```
and remove None comparison for `save_load_path` below